docs/concepts/storage/volumes.md

```diff
@@ -171,6 +172,43 @@ writers simultaneously.
 
 See the [CephFS example](https://github.com/kubernetes/examples/tree/{{page.githubbranch}}/staging/volumes/cephfs/) for more details.
 
+### configMap
+
+The [`configMap`](/docs/tasks/configure-pod-container/configmap/) resource
+provides a way to inject configuration data into Pods.
+The data stored in a `ConfigMap` object can be referenced in a `configMap` type
+of volume and then consumed by containerized applications running in a Pod.
+
+When referencing a `configMap` object, you can simply provide its name in the
+volume to reference it. You can also customize the path to use for a specific
+entry in the ConfigMap.
+For example:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: configmap-pod
+spec:
+  containers:
+    - name: test
+      image: busybox
+      volumeMounts:
+        - name: config-vol
+          mountPath: /etc/config
+  volumes:
+    - name: config-vol
+      configMap:
+        name: log-config
+        items:
+          - key: log_level
+            path: log_level
+```
+
+The Pod created using the above manifest references a ConfigMap object in the
```

Comment:
If you make the change suggested above, you can probably take out the first sentence and instead have something like:

> The `log-config` ConfigMap is mounted as a volume, and all contents stored in its `log_level` entry is mounted into the Pod at path `/etc/config/log_level`. (Note that this path is derived from the volume's `mountPath` and the `path` keyed with `log_level`).

Reply:
sounds good